src/index.ts

```diff
@@ -245,14 +250,18 @@ class Marian {
     index: Index
     client: MongoClient
     db: Db
-    collection: Collection
+    lastRefresh: RefreshInfo | null
+    documents: Collection
+    manifests: Collection
```

Comment:
does marian store the manifests in a collection of a mongo database currently?

Reply:
if so, how have I never seen it before 😃

Reply:
Marian has no interaction with mongodb -- its index is entirely internal.

(also, I actually may remove this collection, since it's only ever written to)